src/operator/api/v1alpha1/intents_types.go

```diff
@@ -166,35 +167,39 @@ func (in *ClientIntents) GetIntentsLabelMapping(requestNamespace string) map[str
 	otterizeAccessLabels := map[string]string{}
 
 	for _, intent := range in.GetCallsList() {
-		ns := intent.ResolveIntentNamespace(requestNamespace)
+		ns := intent.ResolveServerNamespace(requestNamespace)
 		formattedOtterizeIdentity := GetFormattedOtterizeIdentity(intent.Name, ns)
 		otterizeAccessLabels[fmt.Sprintf(OtterizeAccessLabelKey, formattedOtterizeIdentity)] = "true"
 	}
 
 	return otterizeAccessLabels
 }
 
-// ResolveIntentNamespace returns target namespace for intent if exists
+// ResolveServerNamespace returns target namespace for intent if exists
 // or the entire resource's namespace if the specific intent has no target namespace, as it's optional
-func (in *Intent) ResolveIntentNamespace(intentsObjNamespace string) string {
+func (in *Intent) ResolveServerNamespace(intentsObjNamespace string) string {
 	if in.Namespace != "" {
 		return in.Namespace
 	}
 
 	return intentsObjNamespace
 }
 
+func (in *Intent) typeAsGQLType() graphqlclient.IntentType {
+	return graphqlclient.IntentType(strings.ToUpper(string(in.Type)))
```

Comment:
Please map it manually - this is just waiting to turn into a bug. Use the type system